.appveyor.yml

```diff
@@ -65,7 +65,7 @@ install:
 
   # Install requirements from inside conda environment
   - cmd: activate testenv
-  - cmd: conda install -c conda-forge fbprophet  # prior fbprophet installation
+  - cmd: conda install -c conda-forge "matplotlib>=3.3.2" "seaborn>=0.11.0" "tsfresh>=0.17.0" "hcrystalball>=0.1.9" "stumpy>=1.5.1" "fbprophet>=0.7.1" "numba>=0.50,<0.53" "numpy>=1.19" "pandas>=1.1.0,<1.2" "statsmodels>=0.12.1" "scikit-learn>=0.23.0" "scikit-posthocs>=0.6.5" pystan
```

Comment:
Can we not install from `%REQUIREMENTS`, i.e. `build_tools/requirements.txt` here? We're already specifying our dependencies in way to many places - don't want to add another one here! 🤔

Reply:
The problem is that not all requirements are available from `conda-forge` (yet). So you'd have to also maintain a `conda` `environment.yml` or inline it as above.

*Aside:* In general `requirements.txt` won't always work as `conda` packages can have different names to their `pip`  counterparts.

Reply:
☝️ I'm not too sure what can be done about that - I can't see any *good* options. Probably a separate `environment.yml` is easier to maintain (more visible) than inlining the deps and constraints here.

Reply:
Yes, okay. Should we try to move away from `requirements.txt` and to `environment.yml` where you can separately specify pip installables?

Reply:
Probably best to do that in a separate PR though to merge this one first

Reply:
That wouldn't work for `pip` installing though. Eventually when all deps are on CF you wouldn't need to maintain two files. I'm happy to create the `environment.yml` but that will have to wait until tomorrow night. That can easily be done in a separate PR though, if you want to merge this to fix the Access Violation on master.

Reply:
Yeah let's do that! Re the env file, I have something like [this](https://github.com/heidelbergcement/hcrystalball/blob/25f186dc72d4e273c6696a5c822f601d54bab734/environment.yml#L28) in mind where you specify pip installs in the conda env file.

Reply:
That would work for *`conda`* installing the deps but you wouldn't be able to *`pip`* install them from that file. If you don't care about `pip` installing the deps then that would be the way to go.